connectors/google/google-sheets/src/main/java/io/camunda/connector/gsheets/model/request/impl/AddValues.java

```diff
@@ -60,54 +58,4 @@ public Object getValue() {
   public void setValue(Object value) {
     this.value = value;
   }
-
-  @Override
-  public String getType() {
-    return type;
-  }
-
-  @Override
-  public void setType(String type) {
-    this.type = type;
-  }
-
-  @Override
-  public boolean equals(Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    AddValues addValues = (AddValues) o;
-    return Objects.equals(spreadsheetId, addValues.spreadsheetId)
-        && Objects.equals(worksheetName, addValues.worksheetName)
-        && Objects.equals(cellId, addValues.cellId)
-        && Objects.equals(value, addValues.value)
-        && Objects.equals(type, addValues.type);
-  }
-
-  @Override
-  public int hashCode() {
-    return Objects.hash(spreadsheetId, worksheetName, cellId, value, type);
-  }
-
-  @Override
-  public String toString() {
```

Comment:
Shall we keep `equals`, `hashCode`, `toString`?

Reply:
@igpetrov Good point. Please take another look.